webapp/components/post_view/components/post_list.jsx

```diff
@@ -392,15 +392,15 @@ export default class PostList extends React.Component {
         if (this.props.scrollType === ScrollTypes.BOTTOM) {
             this.scrollToBottom();
         } else if (this.props.scrollType === ScrollTypes.NEW_MESSAGE) {
-            window.setTimeout(window.requestAnimationFrame(() => {
+            window.requestAnimationFrame(() => {
```

Comment:
The only thing that worries me with this approach is that the function inside the `requestAnimationFrame` could be could multiple times. I don't remember if its something like 60 times per second or something crazy like that